modules/extension/wms/src/main/java/org/geotools/data/wms/WMS1_0_0.java

```diff
@@ -25,19 +25,11 @@
 import java.util.Iterator;
 import java.util.Map;
 
-import org.geotools.data.ows.AbstractGetCapabilitiesRequest;
-import org.geotools.data.ows.AbstractRequest;
-import org.geotools.data.ows.GetCapabilitiesRequest;
-import org.geotools.data.ows.HTTPResponse;
-import org.geotools.data.ows.Response;
-import org.geotools.data.wms.request.AbstractGetFeatureInfoRequest;
-import org.geotools.data.wms.request.AbstractGetMapRequest;
-import org.geotools.data.wms.request.DescribeLayerRequest;
-import org.geotools.data.wms.request.GetLegendGraphicRequest;
-import org.geotools.data.wms.request.GetStylesRequest;
-import org.geotools.data.wms.request.PutStylesRequest;
+import org.geotools.data.ows.*;
+import org.geotools.data.wms.request.*;
```

Comment:
I'm using IntelliJ, I agree with you about wildcard, it reduce readability. So turn this feature off.